src/twisted/python/_release.py

```diff
@@ -35,6 +35,7 @@
     u"https://python-hyper.org/h2/en/stable/objects.inv",
     u"https://python-hyper.org/priority/en/stable/objects.inv",
     u"https://zopeinterface.readthedocs.io/en/latest/objects.inv",
+    u"https://automat.readthedocs.io/en/latest/objects.inv",
```

Comment:
no longer necessary I guess.  OTOH Twisted uses automat elsewhere so it's probably a good thing to have.